dotnet/src/Microsoft.Agents.AI.Mem0/Mem0Provider.cs

```
@@ -166,12 +178,14 @@ public override async ValueTask InvokedAsync(InvokedContext context, Cancellatio
             return; // Do not update memory on failed invocations.
         }
 
-        // Persist request and response messages after invocation.
-        await this.PersistMessagesAsync(context.RequestMessages, cancellationToken).ConfigureAwait(false);
-
-        if (context.ResponseMessages is not null)
+        try
+        {
+            // Persist request and response messages after invocation.
+            await this.PersistMessagesAsync(context.RequestMessages.Concat(context.ResponseMessages ?? []), cancellationToken).ConfigureAwait(false);
+        }
+        catch (Exception ex)
         {
-            await this.PersistMessagesAsync(context.ResponseMessages, cancellationToken).ConfigureAwait(false);
+            this._logger?.LogError(ex, "Mem0AIContextProvider: Failed to send messages to Mem0 due to error");
```

Is the exception intentionally swallowed here?